src/commands/run.rs

```diff
@@ -144,13 +144,28 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
         // this is for `rails c` and similar REPLs
     })?;
 
-    let exit_status =
-        tokio::process::Command::new(args.args.first().context("No command provided")?)
-            .args(args.args[1..].iter())
-            .envs(variables)
-            .status()
-            .await
-            .context("Failed to spawn command")?;
+    let mut args = args.args.iter().map(|s| s.as_str()).collect::<Vec<_>>();
+    if args.is_empty() {
+        bail!("No command provided");
+    }
+
+    // So the linter doesn't get mad. I think this is pretty ugly.
+    let child_process_name = match std::env::consts::OS {
+        "windows" => {
+            args.insert(0, "/C");
+            "cmd"
+        }
+        _ => {
+            args.remove(0);
+            args.first().context("No command provided")?
```

Comment:
We are removing the first arg from the args array but then want to return it from the match. This can be changed to just remove the first item and return it

```suggestion
            args.remove(0)
```